File: algorithm/src/main/java/baekjoon/p05597/Main.java
package baekjoon.p05597;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] inputs = new int[28];

        for (int i = 0; i < inputs.length; i++) {
            inputs[i] = sc.nextInt();
        }

        for (int output : solve(inputs)) {
            System.out.println(output);
        }
    }

    private static List<Integer> solve(int[] inputs) {
        List<Integer> ret = new ArrayList<>();
        boolean[] check = new boolean[31];

        for (int input : inputs) {
            check[input] = true;
        }

        for (int i = 1; i < check.length; i++) {
            if (check[i]) {
                continue;
            }

            ret.add(i);
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p05597/Main.kt
package baekjoon.p05597

fun main(args: Array<String>) {
    val inputs = IntArray(28) { readLine()!!.toInt() }
    for (output in solve(inputs)) {
        println(output)
    }
}

private fun solve(inputs: IntArray): List<Int> {
    val ret = ArrayList<Int>()
    val check = BooleanArray(31) { false }

    for (input in inputs) {
        check[input] = true
    }

    for (i in 1 until check.size) {
        if (check[i]) {
            continue
        }

        ret.add(i)
    }

    return ret
}